src/pretalx/person/tasks.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from django.core.files import File
+from django.core.files.temp import NamedTemporaryFile
+from django.dispatch import receiver
+from requests import get
+
+from pretalx.celery_app import app
+from pretalx.common.signals import periodic_task
+from pretalx.person.models.user import User
+
+logger = logging.getLogger(__name__)
+
+
+@app.task()
+def gravatar_cache(person_id: int):
+    user = User.objects.filter(pk=person_id).first()
+
```

Comment:
Due to race conditions, I think it would be good to filter by `get_gravatar=True` here, or to check that it's still `True`.

(Consider the situation when updating pretalx to run this code: `runperiodic` runs every 1-5 minutes, but a larger instance would have more gravatar users than could be fetched in that time, resulting in duplicate fetches due to `refetch` being triggered again while one set of `refetch` is already running.)